Makefile

```diff
@@ -28,66 +29,72 @@ all: build
 lint:
 	golangci-lint run -v
 
-.PHONY: build
-build: lint unit-test build-minimal generate-keys
-
-build-minimal: create-build-dir
-	CGO_ENABLED=0 go build -o $(BUILD_FOLDER)/assisted-service cmd/main.go
-
-create-build-dir:
+$(BUILD_FOLDER):
 	mkdir -p $(BUILD_FOLDER)
 
 format:
 	goimports -w -l cmd/ internal/ subsystem/
 
+############
+# Generate #
+############
+
 generate:
 	go generate $(shell go list ./... | grep -v 'assisted-service/models\|assisted-service/client\|assisted-service/restapi')
 
-generate-from-swagger:
-	rm -rf client models restapi
+generate-from-swagger: generate-go-client generate-go-server
+
+generate-go-server:
+	rm -rf restapi
```

Comment:
Note that you can replace this with:
```
docker build -f Dockerfile.assisted-service --target=swagger_go
```
swagger_go stage will need a bit of refactor (currently generate both client and server).
I wouldn't do it in this PR but keep in mind.

Reply:
Cool didn't know about this feature.